test/v5/createAmendUkPropertyCumulativeSummary/def1/model/request/ExpensesSpec.scala

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v5.createAmendUkPropertyCumulativeSummary.def1.model.request
+
+import play.api.libs.json.{JsValue, Json}
+import support.UnitSpec
+
+class ExpensesSpec extends UnitSpec {
+
+  val consolidatedDownstreamRequest: JsValue = Json.parse(
+    """{
+      |  "consolidatedExpenses": 4.17,
+      |  "residentialFinancialCostAmount": 4.18,
+      |  "broughtFwdResidentialFinancialCostAmount": 4.20,
+      |  "ukOtherRentARoom": {
+      |      "amountClaimed": 4.21
+      |  }
+      |}
+    """.stripMargin
+  )
+
+  val consolidatedVendorRequestJson: JsValue = Json.parse(
+    """
+      |{
+      | "consolidatedExpenses": 4.17,
+      | "residentialFinancialCost": 4.18,
+      | "residentialFinancialCostsCarriedForward": 4.20,
+      | "rentARoom": {
+      |    "amountClaimed": 4.21
+      |  }
+      |}
+    """.stripMargin
+  )
+
+  val fullExpensesVendorRequestJson: JsValue = Json.parse(
+    """{
+      |  "premisesRunningCosts": 4.11,
+      |  "repairsAndMaintenance": 4.12,
+      |  "financialCosts": 4.13,
+      |  "professionalFees": 4.14,
+      |  "costOfServices": 4.15,
+      |  "other": 4.16,
+      |  "residentialFinancialCost": 4.18,
+      |  "travelCosts": 4.19,
+      |  "residentialFinancialCostsCarriedForward": 4.20,
+      |  "rentARoom": {
+      |    "amountClaimed": 4.21
+      |  }
+      |}
+    """.stripMargin
+  )
+
+  val fullExpensesMtdRequestJson: JsValue = Json.parse(
```

Comment:
Maybe rename `fullExpensesDownstreamRequest` to make it clear that it is for downstream not for MTD.